catchpoint-mixin/panels.libsonnet

```diff
@@ -9,6 +9,110 @@ local utils = commonlib.utils;
       local stat = g.panel.stat,
       local barGauge = g.panel.barGauge,
       local fieldOverride = g.panel.table.fieldOverride,
+      local alertList = g.panel.alertList,
+      topAvgLoadTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by test name',
+          targets=[t.topAvgLoadTimeTestName],
+          description='Average total time it took to load the webpage by test name, helping assess the responsiveness and speed of the monitored webpage.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
 
+      topMaxTotalLoadTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max total load time',
+          targets=[t.topMaxTotalLoadTime],
+          description='Maximum total time it took to load the webpage by test name, indicating the worst-case scenario for webpage loading performance.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by test name',
+          targets=[t.topAvgDocumentCompletionTimeTestName],
+          description='Average time taken for the browser to fully render the page after all resources are downloaded by test name, serving as a key indicator of end-user perceived load time.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topMaxDocumentCompletionTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max document completion time',
+          targets=[t.topMaxDocumentCompletionTime],
+          description='Maximum time taken for the browser to fully render the page after all resources are downloaded by test name, highlighting the worst-case scenario for end-user perceived load time.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgRequestRatioTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average request ratio by test name',
+          targets=[t.topAvgRequestSuccessRatioTestName, t.topAvgRequestFailureRatioTestName],
+          description='Average ratio of successful and failed requests by test name.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit'),
+
+      topMaxRequestRatioTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max request ratio by test name',
```

Comment:
I think this is supposed to be the `Top max request success vs. failures ratio`. It is not by test name.